frontend/src/components/Nav/Masthead/UserDropdown.tsx

```diff
@@ -93,6 +98,16 @@ class UserDropdownComponent extends React.Component<UserProps, UserState> {
     }
   };
 
+  handleTheme = () => {
+    if (this.state.theme === themes[0]) {
+      this.setState({ theme: themes[1] });
+      document.getElementsByTagName('html')[0].classList.add(darkmode);
```

Comment:
HTML can be retrieved by `document.documentElement` (https://developer.mozilla.org/en-US/docs/Web/API/Document/documentElement)